Tests/js/views/services/assets/ez-locationviewviewservice-tests.js

```diff
@@ -430,6 +430,191 @@ YUI.add('ez-locationviewviewservice-tests', function (Y) {
             Y.Mock.verify(app);
             Y.Mock.verify(contentMock);
         },
+
+        "Should launch the universal discovery widget when receiving an copyAction event": function () {
+            this.service.on('contentDiscover', function (e) {
+                Y.Assert.isObject(e.config, "contentDiscover config should be an object");
```

Comment:
you need to add a boolean flag, otherwise if the contentDiscover event is not fired, the test will still pass.